lib/utilities/globals.dart

```diff
@@ -0,0 +1,3 @@
+import 'package:flutter/material.dart';
+
+final GlobalKey<ScaffoldMessengerState> snackBarKey = GlobalKey<ScaffoldMessengerState>();
```

Comment:
no need for this, just directly place this above ur `PokemonOverviewPage` but don't forget to make it private 🙂